hack/update-codegen.sh

```diff
@@ -34,3 +34,10 @@ bash "${CODEGEN_PKG}"/generate-groups.sh "client" \
   --fake-clientset=false \
   --output-base "$SCRIPT_ROOT/../../../" \
   --go-header-file "${SCRIPT_ROOT}"/hack/boilerplate.go.txt
+
+bash "${CODEGEN_PKG}"/generate-groups.sh "client" \
```

Comment:
We need to use generic client for kubevirt in the future. It is not the issue with CDI but with kubevirt.